iotdb/src/main/java/org/apache/iotdb/db/engine/filenode/FileNodeProcessor.java

```diff
@@ -1706,7 +1726,7 @@ public FileNodeFlushFuture flush() throws IOException {
     if (bufferWriteProcessor != null && !bufferWriteProcessor.isClosed()) {
       bufferWriteFlushFuture = bufferWriteProcessor.flush();
     }
-    if (overflowProcessor != null && !overflowProcessor.isClosed()) {
+    if (overflowProcessor != null && !bufferWriteProcessor.isClosed()) {
```

Comment:
Why change it from overflowProcessor to bufferWriteProcessor?

Reply:
=__=!! it is a bug...